akka-http-core/src/test/scala/akka/http/impl/engine/ws/WithMaterializerSpec.scala

```diff
@@ -13,7 +13,8 @@ import akka.testkit.TestKit
 trait WithMaterializerSpec extends BeforeAndAfterAll { _: Suite ⇒
   lazy val testConf: Config = ConfigFactory.parseString("""
   akka.event-handlers = ["akka.testkit.TestEventListener"]
-  akka.loglevel = WARNING""")
+  akka.loglevel = DEBUG
```

Comment:
Maybe not leave this in? Especially when there is the keep-all-logs-on-failure-but-hide-on-success-feature somewhere in Akka HTTP

Reply:
ok